src/rouge/core/workflow/steps/fetch_patch.py

```
@@ -0,0 +1,83 @@
+                raw={
+                    "patch_id": patch.id,
+                    "issue_id": issue_id,
+                    "description": patch.description,
+                },
+                adw_id=context.adw_id,
+            )
+
+            return StepResult.ok(None)
+
+        except ValueError as e:
+            logger.error(f"Error fetching patch: {e}")
```

Ruff (0.14.11)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)